addons/mail/models/ir_attachment.py

```diff
@@ -75,11 +75,13 @@ def _attachment_format(self, legacy=False):
         res_list = []
         for attachment in self:
             res = {
-                'checksum': attachment.checksum,
-                'id': attachment.id,
-                'filename': attachment.name,
-                'name': attachment.name,
-                'mimetype': 'application/octet-stream' if safari and attachment.mimetype and 'video' in attachment.mimetype else attachment.mimetype,
+                "checksum": attachment.checksum,
+                "id": attachment.id,
+                "filename": attachment.name,
+                "name": attachment.name,
+                "mimetype": "application/octet-stream" if safari and attachment.mimetype and "video" in attachment.mimetype else attachment.mimetype,
+                "type": attachment.type,
+                "url": attachment.url,
```

Comment:
I would not change the single quotes to double quotes, this diffs more than necessary, which just makes reviewing the relevant changes harder, and makes fw-port more prone to conflicts and harder to adapt.

Really such changes should be limited to master.

Reply:
ok, that was generated by the formatting tools. I agree that this should only be in master. should I do it when fw to master or just leave it there until next time we change the code?

Reply:
Better have minimal diff for FIX, so the FW process is the smoothest. Code-cleaning considerations are secondary, nice-to-have if they pose no issue (= changes on LOCs already in diff)